src/lib/mod.rs

```diff
@@ -107,6 +116,24 @@ pub mod tests {
         let result = is_gzip_path(&file_path);
         assert_eq!(result, expected);
     }
+
+    // ############################################################################################
+    // Tests is_zstd_path()
+    // ############################################################################################
+
+    #[rstest]
+    #[case("test_fastq.fq", false)] // .fq is invalid zstd
+    #[case("test_fastq.fq.gz", false)] // .fq.gz is invalid zstd
+    #[case("test_fastq.fq.bgz", false)] // .fq.bgz is invalid zstd
+    #[case("test_fastq.fq.tar", false)] // .fq.tar is invalid zstd
+    #[case("test_fastq.fq.zst", true)] // .fq.zst is valid zstd
+    fn test_is_zstd_path(#[case] file_name: &str, #[case] expected: bool) {
+        let dir = TempDir::new().unwrap();
+        let file_path = dir.path().join(file_name);
+        let result = is_zstd_path(&file_path);
+        assert_eq!(result, expected);
+    }
```

Comment:
move to the fgoxide repo?

Reply:
Based on the discussion above, perhaps it is better to move all these "path" tests to fgoxide

Reply:
If possible, that'd be great!

Reply:
Done